auth-helper/src/jwt.rs

```diff
@@ -27,52 +27,65 @@ pub enum Error {
 pub struct Claims {
     /// Issuer.
     /// Identifies the principal that issued the JWT. The processing of this claim is generally application specific.
-    iss: String,
+    pub iss: String,
     /// Subject.
     /// Identifies the principal that is the subject of the JWT. The claims in a JWT are normally statements about the
     /// subject. The subject value MUST either be scoped to be locally unique in the context of the issuer or be
     /// globally unique. The processing of this claim is generally application specific.
-    sub: String,
+    pub sub: String,
     /// Audience.
     /// Identifies the recipients that the JWT is intended for. Each principal intended to process the JWT MUST
     /// identify itself with a value in the audience claim. If the principal processing the claim does not identify
     /// itself with a value in the "aud" claim when this claim is present, then the JWT MUST be rejected. The
     /// interpretation of audience values is generally application specific.
-    aud: String,
+    pub aud: String,
     /// Expiration Time.
     /// Identifies the expiration time on or after which the JWT MUST NOT be accepted for processing. The processing of
     /// the "exp" claim requires that the current date/time MUST be before the expiration date/time listed in the "exp"
     /// claim. Implementers MAY provide for some small leeway, usually no more than a few minutes, to account for clock
     /// skew.
     #[serde(skip_serializing_if = "Option::is_none")]
-    exp: Option<u64>,
+    pub exp: Option<u64>,
     /// Not Before.
     /// Identifies the time before which the JWT MUST NOT be accepted for processing. The processing of the "nbf" claim
     /// requires that the current date/time MUST be after or equal to the not-before date/time listed in the "nbf"
     /// claim. Implementers MAY provide for some small leeway, usually no more than a few minutes, to account for clock
     /// skew.
-    nbf: u64,
+    pub nbf: u64,
     /// Issued At.
     /// Identifies the time at which the JWT was issued. This claim can be used to determine the age of the JWT.
-    iat: u64,
+    pub iat: u64,
 }
 
 impl Claims {
     /// Creates a new set of claims.
-    fn new(iss: String, sub: String, aud: String, nbf: u64) -> Self {
+    pub fn new(iss: impl Into<String>, sub: impl Into<String>, aud: impl Into<String>) -> Self {
```

Comment:
I prefer `impl Into<String>` because it takes ownership instead of cloning in case you're passing a `String` already. I only used `impl ToString` in the `Validation` fns because the underlying type does. :shrug: